### ISSUE TITLE
higher speed or forward inference time?

### ISSUE THREAD

**Issue by Zeeshan313** [Tramac/Fast-SCNN-pytorch#24]
Sorry for asking same question as it is asked before but not answered.
i am interested about the speed 123 fps mentioned by the author for high resolution images. i have trained this network for 1000 epochs and got 94% pixel accuracy and 63.5% mIoU. But i got only 10 fps on GeForce RTX 2080 with resolution of (1024x2048). can you please share your speed.

**Comment by JoaoCH** [comment]
first, thank you very much for making your work available here on github.

I am also not getting a good speed and is far from the 140 frames of the paper.

My inference time is about 10 fps and I have an RTX 2060.

**Comment by GEMhust** [comment]
Don't make predictions in the demo.py file, this speed is inaccurate because the model is reloaded for every prediction. We can refer to the eval.py file, which loads the model only once when predicting images in test. After my modification, I got 40FPS in my laptop with GeForce MX150.
![1](https://user-images.githubusercontent.com/94061005/181678479-343309bb-986d-410d-8d6c-f2616fe7f052.png)
